android/src/main/java/com/tailscale/ipn/IPNService.kt

```diff
@@ -114,12 +115,24 @@ open class IPNService : VpnService(), libtailscale.IPNService {
     }
     b.setUnderlyingNetworks(null) // Use all available networks.
 
-    // Prevent certain apps from getting their traffic + DNS routed via Tailscale:
-    // - any app that the user manually disallowed in the GUI
-    // - any app that we disallowed via hard-coding
-    for (disallowedPackageName in UninitializedApp.get().disallowedPackageNames()) {
-      Log.d(TAG, "Disallowing app: $disallowedPackageName")
-      disallowApp(b, disallowedPackageName)
+    val includedPackages: List<String> =
+        MDMSettings.includedPackages.flow.value?.split(",") ?: emptyList()
```

Comment:
You should trim these strings too (remove whitespace and newlines), in case somebody tosses in a space or a linefeed.